pkg/components/helpers/cache/expirable_lru_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package cache
+
+import (
+	"testing"
+	"testing/synctest"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const testTTL = 5 * time.Minute
```

Comment:
Renaming to the value this contains helps understanding where it is referenced below.

```suggestion
const fiveMin = 5 * time.Minute
```